src/pages/Cheon/Login/CheonLogin.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import './CheonLogin.scss';
+import { useNavigate } from 'react-router-dom';
+
+const CheonLogin = () => {
+  const [emailInput, setEmailIdInput] = useState('');
+  const [pwInput, setPwInput] = useState('');
+  const [disabled, setdisabled] = useState(true);
+  const navigate = useNavigate();
+
+  const handleIdInput = event => {
+    setEmailIdInput(event.target.value);
+  };
+  const handlePwInput = event => {
+    setPwInput(event.target.value);
+  };
```

Comment:
```suggestion
  const getUserInput = event => {
    const { name, value } = event.target;
    setUserInput({ ...userInput, [name]:value });
  };
```
- 계산된 속성명을 활용해도 좋을 것 같습니다.